src/components/card/card.scss

```diff
@@ -39,6 +40,20 @@ $_cover-gap: ms.step(1, 1rem);
 
 $_focus-overflow: (size.$edge-large * -1);
 
+/// The cloudy modifier adds clouds to the bottom right corner and also applies
+/// some fluid padding to accommodate. The clouds feel best when they are sized
+/// just a _tad_ larger than this fluid padding.
+///
+/// @type Number
+/// @access private
```

Comment:
What are these two lines for?

Reply:
Part of the Sassdoc comment, specifying what the variable is for, what it's expected to be and whether or not it's intended for export via `@use`. This is consistent with other variables in this file.

At one time I had experimented with importing Sassdoc comments into our Storybook docs, which I still think is a neat idea, but a challenging one to prioritize. Now I'm just maintaining it for consistency here.